x-pack/plugins/endpoint/server/routes/alerts/alerts.test.ts

```diff
@@ -103,22 +77,37 @@ describe('test alerts route', () => {
     expect(routeConfig.options).toEqual({ authRequired: true });
     expect(mockResponse.ok).toBeCalled();
     const alertResultList = mockResponse.ok.mock.calls[0][0]?.body as AlertResultList;
+
+    // NOTE: only check total in this test, as it's coming directly from the mocked ES
+    // response, and will be unreliable when filtering.
     expect(alertResultList.total).toEqual(21);
+
     expect(alertResultList.request_page_index).toEqual(0);
     expect(alertResultList.result_from_index).toEqual(0);
     expect(alertResultList.request_page_size).toEqual(10);
   });
 
-  it('should return alert results according to pagination params -- POST', async () => {
+  it('should not support POST requests for querying', async () => {
+    try {
+      [routeConfig, routeHandler] = routerMock.post.mock.calls.find(([{ path }]) =>
+        path.startsWith('/api/endpoint/alerts')
+      )!;
```

Comment:
Do we need a `!` here?

Reply:
I believe so... we're just asserting that that path will definitely be found. If we're wrong, the test will fail, so I think this is probably fine? Let me know if you feel strongly... it's the pattern that had already been established, so I was just following it.